paddle/contrib/inference/paddle_inference_api_impl.cc

```diff
@@ -241,10 +241,11 @@ bool NativePaddlePredictor::GetFetch(
     }
 
     outputs->at(i).shape = shape;
-    outputs->at(i).data.length = sizeof(float) * data.size();
-    outputs->at(i).data.data = malloc(outputs->at(i).data.length);
-    std::memcpy(
-        outputs->at(i).data.data, data.data(), outputs->at(i).data.length);
+    auto &buffer = outputs->at(i).data;
+    if (buffer.empty() || buffer.length() < sizeof(float) * data.size()) {
```

Comment:
Maybe we don't want to allow ```buffer.length() > sizeof(float) * data.size()```? If that's the case, how does client know how many bytes to use?

Reply:
It is safe to `Resize` here.

If the client doesn't know how much to allocate or `Resize`, just init with a large enough memory, and do not resize it.